docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3'
+
+services:
+  pg:
+    image: postgres:13-alpine
+    environment:
+      POSTGRES_USER: ckb
+      POSTGRES_PASSWORD: nervos
```

Comment:
This should not be specified directly since forked repo will use them probably.

Reply:
this docker-compose.yml file is used mainly for development.